asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -421,3 +422,18 @@ def time_groupby_count(self, *args, **kwargs):
 
     def time_groupby_sum(self, *args, **kwargs):
         execute(self.df.groupby(by=self.groupby_columns).sum())
+
+
+class TimeGroupByMultiColumn(BaseTimeGroupBy):
+    param_names = ["shape", "ngroups", "groupby_ncols"]
+    params = [
+        UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
+        GROUPBY_NGROUPS[ASV_DATASET_SIZE],
+        [6],
+    ]
+
+    def time_groupby_agg_mean(self, *args, **kwargs):
```

Comment:
please also add `groupby_sum` to the `MultiColumn` benchmark, so we can see performance differences with its equivalent in a single-column groupby. 

"mean" and "nunique" will default to pandas for some time after #3250 will be merged, so their time would be kinda irrelevant to track the performance of multi-column groupby

Reply:
`groupby_sum` case is added